### ISSUE TITLE
Duplicate index creation

### ISSUE THREAD

**Issue by david-alm** [piccolo-orm/piccolo#36]
Tables containing indices trigger a `DuplicateTableError` even when passed the `if_not_exists=True` argument.

```
class Example(Table):
    id = Integer(primary_key=True)
    value = Varchar(length=255, index=True)
 
# Initial table creation
await Example.create_table(if_not_exists=True)
# Duplicate table creation should be idempotent
await Example.create_table(if_not_exists=True)
```

This causes the following error:
`asyncpg.exceptions.DuplicateTableError: relation "example_value" already exists`

**Comment by dantownsend** [comment]
@david-alm Hi David, thanks for reporting this.

I've been able to replicate this error. It should be a pretty simple fix, so will have it fixed soon.

**Comment by dantownsend** [comment]
@david-alm It should be fixed now - version 0.14.12 on PyPI, thanks.
